test/browser/features/page-load-spans.feature

```diff
@@ -6,113 +6,113 @@ Feature: Page Load spans
         And I click the element "stop-clock"
         And I wait to receive 1 trace
 
-        Then a span named "[FullPageLoad]/page-load-spans/" contains the attributes:
-            | attribute                         | type             | value                     |
-            | bugsnag.span.category             | stringValue      | full_page_load            |
-            | bugsnag.browser.page.title        | stringValue      | New title                 |
-            | bugsnag.browser.page.route        | stringValue      | /page-load-spans/         |
+        Then a span named "[FullPageLoad]/docs/page-load-spans/" contains the attributes:
```

Comment:
Example of the above comment here. `"Given I navigate to the test URL "/page-load-spans"` yet I end up with a `"span named [FullPageLoad]/docs/page-load-spans/"`